examples/bitwise.code

```diff
@@ -91,4 +91,12 @@ void main() {
     printint(arr[1]);
 
     printchar('\n');
+
+    print("Order of Execution Test: ");
+    x = 1 | 2 ^ 3 & 5;
+
+    if(x != (1 | (2 ^ (3 & 5)))) {
```

Comment:
did you mean
if (x != 3) 
?

Reply:
It's to better show the order it's parsed